_rules/heading-descriptive-b49b2e.md

```diff
@@ -55,8 +45,11 @@ Implementation of [Presentational Roles Conflict Resolution][] varies from one b
 
 ## Background
 
+Visible headings that are not in the accessibility tree are a failure of [Success Criterion 1.3.1 Info and Relationships][sc131]. These are not tested by this rule but they can still fail [Success Criterion 2.4.6 Headings and Labels][sc246]. 
```

Comment:
I think this is saying that visible headings that are not in the accessibility tree are not tested in this rule, but they are. Perhaps:

```suggestion
Visible headings that are not in the accessibility tree are a failure of [Success Criterion 1.3.1 Info and Relationships][sc131]. These can also fail [Success Criterion 2.4.6 Headings and Labels][sc246]. 
```